### ISSUE TITLE
a11y: Classifiers

### ISSUE THREAD

**Issue by woodruffw** [pypa/warehouse#6082]
On `/classifiers/`:

* Each classifier's "copy to clipboard" link (see screenshot) should be made a button and enlarged, for consistency with other "copy to clipboard" actions. Enlargement also aides users with limited mobility.

<img width="461" alt="Screen Shot 2019-06-24 at 12 16 31 PM" src="https://user-images.githubusercontent.com/3059210/60035044-eba24400-9679-11e9-9af7-2fa8d801dcb8.png">
